src/components/Integrations/Shopify/Shopify.js

```diff
@@ -51,11 +53,20 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
           </tr>
         ) : (
           <tr className="sync">
-            <td>{list.name}</td>
-            <td className="text-sync">
-              <span className="ms-icon icon-clock"></span>
-              {_('common.synchronizing')}
-            </td>
+            {list.state !== SubscriberListState.notAvailable ? (
+              <>
+                <td>{list.name}</td>
+                <td className="text-sync">
+                  <span className="ms-icon icon-clock"></span>
+                  {_('common.synchronizing')}
+                </td>
+              </>
+            ) : (
+              <>
+                <td className="text-sync">{_('shopify.no_list_available')}</td>
+                <td className="text-sync"></td>
```

Comment:
empty column??

Reply:
yes, what do you suggest? leave one column with column span?

Reply:
you can delete

Reply:
As we've talked in slack I will remove this extra column here.